fdbt-dev/sql/operatorGroup.sql

```diff
@@ -0,0 +1,12 @@
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+LOCK TABLES `operatorGroup` WRITE;
+
+TRUNCATE TABLE `operatorGroup`;
+
+INSERT INTO `operatorGroup` (name, contents, nocCode) 
+VALUES
+('test', '[{"nocCode":"LNUD","name":"The Blackburn Bus Company"},{"nocCode":"NWBT","name": "Pilkingtonbus"}]', 'BLAC');
```

Comment:
do we want to add this with the name test if the tests add one with this name?

Reply:
let me change to operator-group